src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,94 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+  const pages = [];
+
+  for (let i = 1; i <= totalPages; i++) {
+    pages.push(i);
+  }
```

Comment:
we can use shorter syntax

```suggestion
  const pages = Array.from({length: totalPages}, (_, i) => i + 1)
```